src/__tests__/config.test.ts

```diff
@@ -50,15 +35,16 @@ describe('compile-json-schema-to-typescript', () => {
 });
 
 describe('validateConfiguration', () => {
-  test('parses minimal configuration', async () => {
+  test('parses minimal configuration', () => {
```

Comment:
Removed this `async` and others because the linter was complaining and nothing in the tests I removed it from was, in fact, async. LMK if it was there for some reason I haven't thought of, and if it needs to go back.